### PR TITLE
Tikit

### PR DESCRIPTION
**Pull request by TikitTeam**
submitting connector for certification

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by sriyen-msft** [comment]
@TikitTeam Please note that we do not review draft PRs, you can delete it from the open-source if you are still working on it.

**Comment by TikitTeam** [comment]
Updated.

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2652 @@
...
+  "consumes": [
+    "multipart/form-data"
+  ],
+  "produces": [],
+  "paths": {
+    "/Ticket": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:27]: @TikitTeam Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2652 @@
...
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:28]: @TikitTeam Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2652 @@
...
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response from AddTicket",
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            }
+          },
+          "default": {
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:121]: @TikitTeam Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2652 @@
...
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:123]: @TikitTeam Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,2652 @@
...
+                  "type": "boolean",
+                  "description": "Closed"
+                },
+                "DueDate": {
+                  "type": "string",
+                  "description": "DueDate"
+                },
+                "ResolutionDate": {
+                  "type": "string",
+                  "description": "ResolutionDate"
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:184]: @TikitTeam All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,2652 @@
...
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Tikit ID",
+            "description": "Enter Tikit ID"
+          }
+        ]
+      }
+    },
+    "/Comment": {},
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:426]: @TikitTeam Empty operations can be removed as they would not add any value to your connector.

File: certified-connectors/Tikit/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "b13c40ee-e073-459e-96b5-3f3cca046a37",
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiProperties.json:8]: @TikitTeam Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

File: certified-connectors/Tikit/apiProperties.json [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "b13c40ee-e073-459e-96b5-3f3cca046a37",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "b13c40ee-e073-459e-96b5-3f3cca046a37",
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiProperties.json:14]: @TikitTeam Please remove the secret value as per Microsoft's security policy. You can replace it with dummy text. `"[[DUMMY]]"`

File: certified-connectors/Tikit/apiProperties.json [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "b13c40ee-e073-459e-96b5-3f3cca046a37"
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiProperties.json:25]: @TikitTeam Please remove the secret value as per Microsoft's security policy. You can replace it with dummy text. "[[DUMMY]]"

File: certified-connectors/Tikit/readme.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+## Tikit
+
+Tikit for Microsoft Teams enables conversational ticketing in a system you already use. Turn messages from chat into tickets, authenticate with M365 single sign on, deflect common questions with artificial intelligence and report with Power BI.
+
+## Prerequisites
+
+You will need to have an active Tikit, [Microsoft 365 Service Desk](https://www.tikit.ai/pricing/) subscription to leverage this connector.
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/readme.md:7]: @TikitTeam Please refer the template provided for certified-connectors and update the readme.md file accordingly. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@TikitTeam Please remove the icon.png and settings.json file from your PR as these files are not required in the open source repo.

File: certified-connectors/Tikit/apiProperties.json (hunk 7 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiProperties.json]: Fixed

File: certified-connectors/Tikit/apiProperties.json (hunk 9 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiProperties.json]: fixex

File: certified-connectors/Tikit/apiProperties.json (hunk 8 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiProperties.json]: fixed

**Comment by sriyen-msft** [comment]
@TikitTeam Please resolve all the comments to get the PR approved and merged.

File: certified-connectors/Tikit/apiDefinition.swagger.json (hunk 2 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiDefinition.swagger.json]: updated

File: certified-connectors/Tikit/apiDefinition.swagger.json (hunk 3 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiDefinition.swagger.json]: updated

File: certified-connectors/Tikit/apiDefinition.swagger.json (hunk 4 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiDefinition.swagger.json]: updated

File: certified-connectors/Tikit/apiDefinition.swagger.json (hunk 5 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiDefinition.swagger.json]: updated

File: certified-connectors/Tikit/readme.md (hunk 10 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/readme.md]: updated

**Comment by TikitTeam** [comment]
Points previously raised have been addressed along with removing settings.json and icon.png

File: certified-connectors/Tikit/apiDefinition.swagger.json [hunk 11]
```diff
@@ -0,0 +1,2031 @@
...
```
Comment (sriyen-msft) [inline certified-connectors/Tikit/apiDefinition.swagger.json:226]: @TikitTeam Empty operations can be removed as they would not add any value to your connector.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Tikit/apiDefinition.swagger.json (hunk 11 above)
Comment (TikitTeam) [reply certified-connectors/Tikit/apiDefinition.swagger.json]: Fixed!

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**